### PR TITLE
Add app version to rules import and version page

### PR DESCRIPTION
**Pull request by jakemco**
The version page is ugly, but it's mostly for debugging. I also can't get the translate resx files to build with my visual studio, so I couldn't add text to go with them even if I wanted to >.<

### CODE REVIEW

File: KillTeam/KillTeam.csproj [hunk 1]
```diff
@@ -387,5 +387,11 @@
     <EmbeddedResource Update="Views\RemerciementPage.xaml">
       <Generator>MSBuild:UpdateDesignTimeXaml</Generator>
     </EmbeddedResource>
+    <EmbeddedResource Update="Views\VersionPage.xaml">
+      <Generator>MSBuild:UpdateDesignTimeXaml</Generator>
+    </EmbeddedResource>
+    <EmbeddedResource Update="Views\View1.xaml">
+      <Generator>MSBuild:UpdateDesignTimeXaml</Generator>
+    </EmbeddedResource>
```
Comment (lacombet) [inline KillTeam/KillTeam.csproj]: What is View1 ?

File: KillTeam.Android/Resources/Resource.designer.cs [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 // <auto-generated>
-//     Ce code a été généré par un outil.
-//     Version du runtime :4.0.30319.42000
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
 //
-//     Les modifications apportées à ce fichier peuvent provoquer un comportement incorrect et seront perdues si
-//     le code est régénéré.
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
```
Comment (lacombet) [inline KillTeam.Android/Resources/Resource.designer.cs:8]: I switched my VS to English so this doesn't happen again ;)

File: KillTeam/Views/VersionPage.xaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+             mc:Ignorable="d"
+             Title="Version"
+             x:Class="KillTeam.Views.VersionPage">
+    <ContentPage.Content>
+        <StackLayout>
+            <StackLayout
+              VerticalOptions="CenterAndExpand"
+              HorizontalOptions="FillAndExpand">
+                <Label Text="{Binding AppVersion}" HorizontalTextAlignment="Center" />
+                <Label Text="{Binding RulesVersion}" HorizontalTextAlignment="Center"/>
```
Comment (lacombet) [inline KillTeam/Views/VersionPage.xaml]: Should we display a label before the version ? 

e.g.

App : [AppVersion]
Db : [DbVersion]

Comment (lacombet) [inline KillTeam/Views/VersionPage.xaml]: Should we truncate the db version ?

File: KillTeam/KillTeam.csproj (hunk 1 above)
Comment (jakemco) [reply KillTeam/KillTeam.csproj]: Oops. It's what happens when you forget to put a name in the new file dialog and then delete it >.<

File: KillTeam/Views/VersionPage.xaml (hunk 3 above)
Comment (jakemco) [reply KillTeam/Views/VersionPage.xaml]: We could, but I can't make the translate Resx files build, so I won't be able to add translations.

Comment (jakemco) [reply KillTeam/Views/VersionPage.xaml]: nah, having the hash is useful in case we ever accidentally repeat a version number.

Comment (lacombet) [reply KillTeam/Views/VersionPage.xaml]: Ok ;) we'll eventually change that when resource management will be cleaner

**Review by lacombet — APPROVED** [review]

**Pull request merged**